runtime/lib/descriptor/activity-descriptor.js

```diff
@@ -410,20 +410,19 @@ Object.assign(ActivityDescriptor.prototype,
         return this._runtime.setForegroundById(this._appId, Object.assign({ form: form }, options)).then(() => {})
       }
     },
-    /**
-     * sync cloudappclient appid stack
-     * @memberof yodaRT.activity.Activity
-     * @instance
-     * @private
-     * @function syncCloudAppIdStack
-     * @param {string[]} stack cloud skills id
-     * @returns {Promise<void>}
-     */
-    syncCloudAppIdStack: {
```

Comment:
Why are the `syncCloudAppIdStack` removed?

Reply:
It's already deprecated in a very early version.